soh/soh/Enhancements/randomizer/3drando/hint_list/hint_list_item.cpp

```diff
@@ -2057,7 +2057,14 @@ void StaticData::HintTable_Init_Item() {
                                                }, {
                                                CustomMessage("a fish-puller", /*german*/"ein Fischzieher", /*french*/"(canne à pêche)")});
                                                 // /*spanish*/(caña de pescar)
-  
+    hintTextTable[RHT_SKELETON_KEY] = HintText(CustomMessage("a Skeleton Key", /*german*/ "ein Universalschlüssel", /*french*/ "un Passe-Partout"),
+                                               // /*spanish*/una Llave Maestra
+                                               {
+                                               CustomMessage("a key", /*german*/ "ein Schlüssel", /*french*/ "une clé")
+                                                // /*spanish*/una llave
+                                               },
+                                               { CustomMessage("a master unlocker", /*german*/ "ein Meisterentsperrer", /*french*/ "un déverrouilleur maître") });
```

Comment:
```suggestion
                                               { CustomMessage("a master unlocker", /*german*/ "ein Meisterentsperrer", /*french*/ "une Clé Passe-Partout") });
```

In french we have an art thing called a Passe-Partout and we need to add the prefix of the key to know we're talking about a key